testing/tests/DevExpress.ui.widgets.dataGrid/columnsResizingReorderingModule.tests.js

```diff
@@ -4448,7 +4477,12 @@ QUnit.module('Headers reordering', {
         const controller = this.createDraggingHeaderViewController();
         const draggingHeader = new TestDraggingHeader(this.component);
 
-        $('#itemsContainer').html('<div style="width:125px; display: inline-block;"></div><div style="width:125px; display: inline-block;"></div>');
+        $('#itemsContainer').html('<div></div><div></div>');
+
+        $('#itemsContainer > div').css({
+            width: '125px',
+            display: 'inline-block'
+        });
```

Comment:
Let's add classes here, instead of setting CSS rules by hand with JS:

```suggestion
        $('#itemsContainer').html(`
          <div class="itemsContainer__child"></div>
          <div class="itemsContainer__child"></div>
        `);
```